commit a62da6f14db79bd7ea435ab095e998b31b3dbb22
Author: Johan Hedberg <johan.hedberg@intel.com>
Date:   Thu Dec 8 08:32:54 2016 +0200

    Bluetooth: SMP: Add support for H7 crypto function and CT2 auth flag
    
    Bluetooth 5.0 introduces a new H7 key generation function that's used
    when both sides of the pairing set the CT2 authentication flag to 1.
    
    Signed-off-by: Johan Hedberg <johan.hedberg@intel.com>
    Signed-off-by: Marcel Holtmann <marcel@holtmann.org>

diff --git a/net/bluetooth/smp.h b/net/bluetooth/smp.h
index ffcc70b6b199..0ff6247eaa6c 100644
--- a/net/bluetooth/smp.h
+++ b/net/bluetooth/smp.h
@@ -57,6 +57,7 @@ struct smp_cmd_pairing {
 #define SMP_AUTH_MITM		0x04
 #define SMP_AUTH_SC		0x08
 #define SMP_AUTH_KEYPRESS	0x10
+#define SMP_AUTH_CT2		0x20
 
 #define SMP_CMD_PAIRING_CONFIRM	0x03
 struct smp_cmd_pairing_confirm {